modules/460-log-shipper/images/vector/Dockerfile

```
@@ -1,11 +1,24 @@
 ARG BASE_UBUNTU
 ARG BASE_RUST
 
-FROM $BASE_RUST as build
+### 1: Build vector
+FROM $BASE_RUST as build-vector
 
 RUN apt-get update \
     && apt-get install -yq \
-      ca-certificates make bash cmake libclang1-9 llvm-9 libsasl2-dev librdkafka-dev protobuf-compiler
+      ca-certificates make bash cmake libclang1-9 llvm-9 libsasl2-dev protobuf-compiler software-properties-common
+
+# Install librdkafka-dev >=2.0 because bundled version (1.9.2) has bugs with CA certificates location.
+# https://github.com/confluentinc/librdkafka/commit/f8830a28652532009e3f16854cb9d5004d9de06b
+RUN apt-get update \
+    && apt-get install -yq \
+      wget build-essential libsasl2-dev libcurl4-openssl-dev libzstd-dev libssl-dev zlib1g-dev libc6-dev unzip \
```

```suggestion
      wget build-essential libcurl4-openssl-dev libzstd-dev libssl-dev zlib1g-dev libc6-dev unzip \
```

libsasl2-dev is dublicated in previous `RUN`

---

I did it intentionally to highlight that this is a dependency for both vector and librdkafka. Do you think it is worth deleting?

---

I think it's okay. Later we can replace one step with the native librdkafka and the other one will keep working